docker/ml.yml

```
@@ -14,5 +14,25 @@ services:
     networks:
       - webnet
 
+  triton:
+    restart: $RESTART_POLICY
+    # This is a custom built of Triton with:
+    # - GRPC/HTTP support
+    # - CPU only (we don't have GPU in production)
+    # - Tensorflow 2 SavedModel and ONNX support
+    # This allows us to reduce significantly the image size
+    # See https://gist.github.com/raphael0202/091e521f2c79a8db8c6e9aceafb6e0b9 for build script
+    image: ghcr.io/openfoodfacts/triton:latest
+    ports:
+      - 8000:8000
+      - 8001:8001
+      - 8002:8002
```

I really would prefer if all port exposure where configurable through .env. 

See https://github.com/openfoodfacts/openfoodfacts-infrastructure/blob/develop/docs/docker.md#enable-configuration-through-environment

```suggestion
      - ${TRITON_EXPOSE_HTTP:-8000}:8000
      - ${TRITON_EXPOSE_GRPC:-8001}:8001
      - ${TRITON_EXPOSE_METRICS:-8001}:8002
```
In this case, it also has the advantage of documenting what ports are doing :blush:

---

It's a good practice indeed, I'm adding this